course/include/isometry.h

```diff
@@ -117,4 +120,49 @@ inline std::ostream &operator<<(std::ostream &ss, const Matrix3 &vec) {
             << vec.row(2)[2] << "]]";
 }
 
+class Isometry {
+public:
+  Isometry() {
+    rotation_ = Matrix3::kIdentity;
+    translation_ = Vector3::kZero;
+  }
+  Isometry(const Vector3 &trans, const Matrix3 &rot) {
+    translation_ = trans;
+    rotation_ = rot;
+  }
+
+  static const Isometry FromTranslation(const Vector3 &vec);
```

Comment:
Why is this `const`? Does it do anything that the return value is `const` here?

Reply:
Not needed and removed